java-symbolic-execution/java-symbolic-execution-plugin/src/test/java/org/sonar/java/se/checks/UnclosedResourcesCheckTest.java

```diff
@@ -34,12 +34,22 @@ void test() {
   }
 
   @Test
-  void test_lombok() {
+  void doesNotRaiseOnLombokCleanupAnnotatedVariable() {
     SECheckVerifier.newVerifier()
       .onFile("src/test/files/se/UnclosedResourcesLombokCheck.java")
       .withCheck(new UnclosedResourcesCheck())
       .withClassPath(SETestUtils.CLASS_PATH)
-      .verifyIssues();
+      .verifyNoIssues();
+  }
+
+  @Test
+  void doesNotRaiseOnLombokCleanupAnnotatedVariable_noSemantic() {
+    SECheckVerifier.newVerifier()
+      .onFile("src/test/files/se/UnclosedResourcesLombokCheck.java")
+      .withCheck(new UnclosedResourcesCheck())
+      .withoutSemantic()
+      .withClassPath(SETestUtils.CLASS_PATH)
```

Comment:
I think we can remove this line as the `withoutSemantic` indicates that we should not pass any library in the class path (the flag is eventually used [here](https://github.com/SonarSource/sonar-java/blob/889e74199f6ffd01bcae78a794352d6a3866a459/java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/internal/InternalCheckVerifier.java#L299)). 
```suggestion
```